client/my-sites/themes/controller.jsx

```diff
@@ -24,6 +24,7 @@ function getProps( context ) {
 	const { basePath, analyticsPageTitle } = getAnalyticsData(
 		context.path,
 		tier,
+		filter,
```

Comment:
This won't work with `getAnalyticsData` as it's currently implemented -- you'd need to change the [implementation](https://github.com/Automattic/wp-calypso/blob/1cf7b7aeb49c2f2a5ebabc1a9faeb6c8316c0127/client/my-sites/themes/helpers.js#L139-L157).